.github/workflows/_produce-data.yaml

```diff
@@ -3,9 +3,18 @@ name: "[internal] Produce data for external analysis"
 on:
   workflow_call:
   workflow_dispatch:
+    inputs:
+      test_workflow_run_id:
+        description: "Unique GitHub workflow run ID to use for data"
+        default: 9605916313
```

Comment:
Is it going to be a problem to set a default if the assumption is always a unique ID?.
might be better to set this as required and not provide a default if so

Reply:
It is a unique ID, though

Reply:
Maybe i'm confused but if you set a default, it will use the default and its hardcoded as 9605916313.
So two runs without an input will have a run_id of 9605916313?

Reply:
Yeah, because this particular run has some corner case attributes that I would like anyone running this (mainly me) to go through if there's no specific test case you wanna test/

Reply:
I see. 
Maybe we can set the default to be some number so it screams out that no id was set.
like 1234567890 or something to make it obvious. I'm not sure how important ID is but it will make sifting through this data easier manually

Reply:
But I don't want to remember what this particular ID which I want to test on majority cases is or have to write it down somewhere, which is why it's a default.

If we do what you're suggesting, we'd have to do that. Are you saying that's a better workflow?

Reply:
nah, it sounds worse if we have to manually remember or write it down for a particular workflow.
Lets leave it as is then